### DISCUSSION TITLE
In which context could be asw_bleeding used?

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#1735] [Q&A]
The current german translation of `asw_bleeding` has something to do with blood loss, but since it's between some wirehack strings it could have something to do with "Entlüftung", that's a german word for [these nouns](https://dict.leo.org/german-english/Entl%C3%BCftung). I mean I would certainly think about something mechanical in conjunction to the wirehack-minigame than something organic loosing blood.

I believe it's not implemented in the current version of the game. But still, I would like to find out in which context it could be useful.

**Comment by BenLubar** [comment]
it being surrounded by hyphens makes me think of https://github.com/ReactiveDrop/reactivedrop_translations/blob/c28db9b1771632111cbd8460e27e14bdc8da0ce5/resource/reactivedrop_english.txt#L359 but it doesn't appear to be used anywhere in the game's code.

maybe something like this that was cut in Valve-era development?

![image](https://github.com/user-attachments/assets/51a49609-862b-4c50-b957-90d1e1f194b8)

there's a comment about a (cut) feature where the game would track kills made while an officer marine was bleeding out in the code but no other references to bleeding I can see that would be from the era of that string
